plugin.video.zdf_de_2016/addon.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.zdf_de_2016" name="ZDF Mediathek 2016" version="1.0.1" provider-name="Generia">
+  <requires>
+    <import addon="xbmc.python" version="2.24.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="addon.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GBb">ZDF Mediathek 2016</summary>
+    <summary lang="de_DE">ZDF Mediathek 2016</summary>
+    <description lang="en_GB">ZDF Mediathek 2016 new variant after zdf.de website relaunch
+    
+See also https://github.com/generia/plugin.video.zdf_de_2016
+    </description>
+    <description lang="de_DE">ZDF Mediathek 2016 neue Variante nach zdf.de Website Relaunch
+    
+Siehe auch https://github.com/generia/plugin.video.zdf_de_2016
+    </description>
+	<disclaimer lang="en">Some parts of this addon may not be legal in your country of residence - please check with your local laws before installing.</disclaimer>
+	<license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
+	<source>https://github.com/generia/plugin.video.zdf_de_2016</source>
+	<forum></forum>
+	<email>alex at generia dot de</email>
+	<website>http://www.zdf.de</website>
+    <platform>all</platform>
+    <news>
+1.0.1 (2016-12-03)
+- added rubrics: Live-TV, Shows A-Z
```

Comment:
Just retested this version on my Raspberry. Works fine. Is the CR issue a blocker?

PS: Background:
I normal develop on OSX. When I checkout the repo-plugins Jarvis branch git shows a bunch of files as modified. These modifications are only due to cr/lf lineendings which git is handling during checkout. There are lots of posts how to remove this, but none actually work for me.

As a last resort I was using an old Windows machine to push the plugin-source (using autocrlf=true).